features/autofill.lua

```diff
@@ -13,20 +13,9 @@ Settings.register(enable_autofill_name, Settings.types.boolean, true, 'autofill.
 Settings.register(ammo_count_name, Settings.types.positive_integer, 10, 'autofill.ammo_count')
 
 local Public = {}
-
-local ammo_locales = {
-    ['uranium-rounds-magazine'] = {'item-name.uranium-rounds-magazine'},
-    ['piercing-rounds-magazine'] = {'item-name.piercing-rounds-magazine'},
-    ['firearm-magazine'] = {'item-name.firearm-magazine'}
-}
-local k2_ammo_locales = {
-    ['rifle-magazine'] = {'item-name.rifle-magazine'},
-    ['armor-piercing-rifle-magazine'] = {'item-name.armor-piercing-rifle-magazine'},
-    ['uranium-rifle-magazine'] = {'item-name.uranium-rifle-magazine'},
-    ['imersite-rifle-magazine'] = {'item-name.imersite-rifle-magazine'}
-}
-if script.active_mods["Krastorio2"] then ammo_locales = k2_ammo_locales end
-Public.ammo_locales = ammo_locales
+local Ammos = require 'autofill_compatibility'
```

Comment:
We have to use full paths for `requires`.
This is a custom Redmew server thing. For each scenario we remove unused files by statically analysing the .lua files for files they require. And that only works if we use full paths and literal strings that can be determined without having to run any lua.

```suggestion
local Ammos = require 'features.autofill_compatibility'
```